<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateAnnouncementDialog</class>
 <widget class="QDialog" name="UpdateAnnouncementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EspINA Update</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../rsc/espina.qrc">
    <normaloff>:/espina/espina.svg</normaloff>:/espina/espina.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>A new version of EspINA is available!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_versionComparison">
     <property name="text">
      <string>EspINA %1 is now available (you have %2). Would you like to update now?</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Release notes:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="m_releaseNotesText">
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_skipButton">
       <property name="statusTip">
        <string>Skip this version and do not report it again.</string>
       </property>
       <property name="text">
        <string>Skip this version</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_laterButton">
       <property name="statusTip">
        <string>Remind me again on the next check.</string>
       </property>
       <property name="text">
        <string>Remind me later</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_updateButton">
       <property name="statusTip">
        <string>Download new version from the EspINA website.</string>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rsc/espina.qrc"/>
 </resources>
 <connections/>
</ui>
